recipes/hilive2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "hilive2" %}
+{% set version = "2.0a" %}
+
+package:
+ name: "{{ name|lower }}"
+ version: "{{ version }}"
+
+source:
+ - folder: hilive2
+   url: https://gitlab.com/rki_bioinformatics/HiLive2/-/archive/v{{ version }}/HiLive2-v{{ version }}.tar.bz2
+   sha256: 47a8e0085ba0c19d672eecca7c5b70af8de55008fb4aec6dba24fa6a1c150137
+ - folder: seqan
+   url: https://github.com/seqan/seqan/archive/seqan-v2.3.2.tar.gz
+   sha256: 73ce4b344e6a259db1518547544c730f548bc0b32292b30e3a027146e158085e
+
+build:
+ number: 0
+ skip: True # [osx]
+
+requirements:
+ build:
+  - {{ compiler('cxx') }}
+  - cmake >=3.10
+ host:
+  - zlib
+  - boost =1.69.0
+  - lz4
+ run:
+  - zlib
+  - boost =1.69.0
+  - lz4
```

Comment:
Are you dynamically linking all of this? Cmake has a tendency to compile everything statically.